src/Apps/Sell/Components/SubmissionHeader.tsx

```diff
@@ -199,3 +210,89 @@ export const SubmissionHeader: React.FC = () => {
     </Sticky>
   )
 }
+
+const HeaderArtsyLogo: React.FC<{ withExitConfirmation }> = ({
+  withExitConfirmation,
+}) => {
+  const { router } = useRouter()
+  const { isLoggedIn } = useSystemContext()
+
+  const [showExitConfirmationModal, setShowExitConfirmationModal] = useState(
+    false
+  )
+
+  const handleClick = () => {
+    withExitConfirmation
+      ? setShowExitConfirmationModal(true)
+      : router.push("/sell")
+  }
+
+  // When logged in, it's possible to easily exit the flow by clicking on "Save & Exit"
+  // and the Artsy logo should not be clickable.
```

Comment:
I think, we could consider making the Artsy logo clickable in any case for a more cohesive user experience. For a logged in user after clicking on the Artsy logo we could save the draft submission and inform the user about it by showing a banner or a dialog. A draft submission would be shown on the Sell page (as per current implementation)

Reply:
This could also be an option. I decided against it to provide as few ways as possible to leave the flow because we want users to finish the form. I think this is the reason why we removed the link from the logo in the first place. And there is still the "Exit" or "Save & Exit" button. 

I'm open to do this change but think we should discuss it first with the team if we consider it.